.github/workflows/cron-streams-active-cleanup.yaml

```diff
@@ -0,0 +1,15 @@
+name: "cron: Active streams cleanup"
+
+on:
+  schedule:
+    - cron: "*/10 * * * *" # run every 10 minutes
+
+jobs:
+  cron:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clean-up active streams
+        run: |
+          curl --request POST \
+          --url 'https://livepeer.studio/api/stream/job/active-cleanup' \
```

Comment:
Will this GH Action fail, if the cleanup fails? I think we would like to have it working this way:
1. The call `https://livepeer.studio/api/stream/job/active-cleanup` would be sync, so the GH Action effectively waits for the streams to get cleaned up
2. If it fails, the whole job should fail, so that we know that something is wrong

Reply:
Good call. I've ran [some tests with the GH action](https://github.com/livepeer/studio/actions/workflows/cron-streams-active-cleanup.yaml) and we can get the action to fail simply by adding. `--fail` to the curl command, so it exits with a bad code if the server responds a bad status as well. 

The clean-up implementation itself is already synchronous as well, so that part should be working once the API actually gets deployed.